docs/content/doc/setup/reverse-proxies.md

```diff
@@ -96,6 +98,115 @@ location ~* ^/(api|dav|\.well-known)/ {
 6. After saving the edited file, return to NPM's UI browser window and refresh the page to verify your Proxy Host for Vikunja is still online. 
 7. Now, switch over to your Vikunja browswer window and hit refresh. If you configured your URL correctly in original Vikunja container, you should be all set and the browser will correctly show Vikunja. If not, you'll need to adjust the address in the top of the login subscreen to match your proxy address.
 
+### 2. Using the GUI
+
+This guide puts everything in a custom docker network so no ports are exposed, if your setup requires otherwise you will have to uncomment them
+
+```
+version: '3'
+
+services:
+  db:
+    image: postgres:13
+    environment:
+      POSTGRES_PASSWORD: secret
+      POSTGRES_USER: vikunja
+    volumes:
+      - /path/to/vikunja/db:/var/lib/postgresql/data
+    restart: unless-stopped
+  api:
+    image: vikunja/api
+    environment:
+      VIKUNJA_DATABASE_HOST: db
+      VIKUNJA_DATABASE_PASSWORD: secret
+      VIKUNJA_DATABASE_TYPE: postgres
+      VIKUNJA_DATABASE_USER: vikunja
+      VIKUNJA_DATABASE_DATABASE: vikunja
+    #ports:
+    #  - 3456:3456
+    volumes: 
+      - /path/to/vikunja/files:/app/vikunja/files
+    depends_on:
+      - db
+    restart: unless-stopped
+  frontend:
+    image: vikunja/frontend
+    #ports:
+    #  - 80:80
+    restart: unless-stopped
+```
+Create a docker-compose.yml file and run `docker-compose up -d`
+
+After it has spun up the containers it assigns them automatically a network `vikunja_default`.
+For this setup to work you must assign the containers on the same network your NPM is.
+If you have ports exposed in the yaml you don't have to.
+
+You can check that by running `docker ps` to find your instance id
+
+![2022-01-06_13-47](https://user-images.githubusercontent.com/76781738/148378509-e56829c9-59d6-4ad5-af13-e94412c2a18e.png)
+
+and then run `docker inspect -f "{{json .NetworkSettings.Networks  }}" $INSTANCE_ID` to find the network name
+
+For example, let's say npm is running on `npm_default`
+To assign the containers execute 
```

Comment:
Wouldn't it be easier to add the npm network directly in the docker-compose config? Similar to [the traefik example](https://vikunja.io/docs/full-docker-example/#example-with-traefik-2) (note the `web` network here).

Reply:
That is true. I think this way it's easier.
I believe it would look like this if you were to add all the containers to an existing network:
```
networks:
  default:
    external: true
    name: my-pre-existing-network
```
But if you then publish the ports your db would also be exposed. That's why in the traefik guide they have 2 networks.
But I think for an NPM setup it becomes too complicated.

Reply:
I wouldn't say this becomes too complicated with two networks, you're pretty much doing the same thing here only manual. Putting everything in the docker-compose config would automate that away.

Reply:
I've never really done it this way, tbh. I will look into it more.